src/gnatss/main.py

```diff
@@ -512,17 +580,13 @@ def load_data(all_files_dict: Dict[str, Any], config: Configuration) -> pd.DataF
         typer.echo(transponder)
     typer.echo("Finished computing harmonic mean")
 
-    # Read deletion file
-    # Set default to empty string
-    all_files_dict.setdefault("deletions", "")
+    # Read deletion file, and set default to empty list
     typer.echo("Load deletions data...")
-    cut_df = load_deletions(all_files_dict["deletions"], config=config)
+    cut_df = load_deletions(all_files_dict.get("deletions", list()), config=config)
```

Comment:
Hmm... we discussed that this should be the following instead:
```python
if all_files_dict.get("deletions"):
    typer.echo("Load deletions data...")
    cut_df = load_deletions(all_files_dict["deletions"], config=config)
```

This way, then even the typer echo is not shown if deletions data is not specified